### ISSUE TITLE
foundry.js:316 Error: This is not a registered game setting

### ISSUE THREAD

**Issue by stgreenb** [mclemente/healthEstimate#51]
Recently upgraded to v9 foundry (fresh install). 

No health estimates showing, and going into the settings of the module I do see the following console error
```
foundry.js:316 Error: This is not a registered game setting
[Detected 3 packages: healthEstimate, colorsettings, lib-wrapper]
    at ClientSettings.get (foundry.js:2115)
    at sGet (utils.js:77)
    at prepSetting (styleSettings.js:66)
    at HealthEstimateStyleSettings.getData (styleSettings.js:73)
    at HealthEstimateStyleSettings._render (foundry.js:3031)
    at HealthEstimateStyleSettings._render (foundry.js:3733)
    at HealthEstimateStyleSettings.render (foundry.js:2990)
    at ExtendedSettingsConfig.onClickSubmenuWrapper (colorSetting.js:20)
    at Z.🎁call_wrapper [as call_wrapper] (libWrapper-wrapper.js:586)
    at ExtendedSettingsConfig.🎁SettingsConfig.prototype._onClickSubmenu#0 (libWrapper-wrapper.js:184)
    at HTMLButtonElement.dispatch (jquery.min.js:2)
    at HTMLButtonElement.v.handle (jquery.min.js:2)
onError @ foundry.js:316
🎁call_wrapped @ libWrapper-wrapper.js:502
🎁Hooks.onError#lib-wrapper @ listeners.js:137
🎁Hooks.onError#0 @ libWrapper-wrapper.js:182
(anonymous) @ foundry.js:2992
Promise.catch (async)
render @ foundry.js:2990
onClickSubmenuWrapper @ colorSetting.js:20
🎁call_wrapper @ libWrapper-wrapper.js:586
🎁SettingsConfig.prototype._onClickSubmenu#0 @ libWrapper-wrapper.js:184
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
```
Relevant Mod versions active

Health Estimate v26.12;
lib - Color Settings v2.8.1;
libWrapper v1.11.0.1;

**Comment by stgreenb** [comment]
Other error detected
```
dnd5e.js:35 Uncaught (in promise) TypeError: t(...).join is not a function
[Detected 1 package: healthEstimate]
    at Module.settings (dnd5e.js:35)
    at systemSpecifics.js:90
settings @ dnd5e.js:35
(anonymous) @ systemSpecifics.js:90
```

**Comment by mclemente** [comment]
Reinstall Health Estimate, 2.6.12 was released as 26.12, so it won't update to 2.6.18.
